api/infrastructure/parallelcluster-api.yaml

```diff
@@ -620,6 +620,8 @@ Resources:
               - lambda:AddPermission
               - lambda:RemovePermission
               - lambda:UpdateFunctionConfiguration
+              - lambda:ListTags
+              - lambda:UntagResource
```

Comment:
Shall we add the policy to `ParallelClusterBuildImageManagedPolicy` for 
`!Sub 'arn:${AWS::Partition}:lambda:${Region}:${AWS::AccountId}:function:ParallelClusterImage-*'`?
Build image also use lambda function.
Same comment for documentation

Reply:
no, because these new policies are for updating a lambda but build image stack cannot be updated